package.json

```diff
@@ -88,7 +88,8 @@
     "react-prism": "^4.3.2",
     "react-scroll": "^1.8.2",
     "rehype-react": "^6.2.0",
-    "simple-dropzone": "^0.8.1"
+    "simple-dropzone": "^0.8.1",
+    "@tweenjs/tween.js": "^18.6.4"
```

Comment:
You should register this lib to  **packages** in *siteconfig.json* fle l, otherwise the codeActions such as *CodePen* and *CodeSandbox* will not work.

Reply:
done